src/test/java/com/thebois/models/world/resources/RockTests.java

```diff
@@ -0,0 +1,49 @@
+package com.thebois.models.world.resources;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class RockTests {
+
+    @Test
+    public void getTypeFromRockResource() {
+        // Arrange
+        final Rock rock = new Rock(1, 1);
+        final ResourceType resourceType;
+
+        // Act
+        resourceType = rock.getType();
```

Comment:
question

Still don't see the point in splitting up the declaration and assignment, how come you've chosen to do that everywhere?

Reply:
Makes the tests unnecessarily long imo.

Reply:
I have no clue why I'm doing it, but I started doing it and then I just tried to be consistent. Will change it and try to get rid of the habbit.